crates/holochain/src/core/workflow/app_validation_workflow/get_zomes_to_invoke_tests.rs

```diff
@@ -883,166 +885,6 @@ async fn store_record_delete_link() {
     assert_matches!(zomes_to_invoke, ZomesToInvoke::OneIntegrity(z) if z.name == zome.name);
 }
 
-// not a logical case that a delete is deleted, but valid nonetheless
-#[tokio::test(flavor = "multi_thread")]
-async fn store_record_delete_of_delete_entry() {
```

Comment:
As a general comment, is there a need to remove these tests? Do they cause harm? Are they particularly slow or flaky?

If this is a flow that we allow in app validation but not in system validation then I like the idea of testing for that. I know, as I've been told in the past, that I can be quite obsessive with testing every possible flow but I'm just wondering.

Reply:
They're not expensive tests amd not flaky at all. I'm fine with keeping them. Mind though that the code path cannot be hit in theory. Sys validation will reject deletes of deletes, so such a delete will never make it to app validation.

Reply:
The point of sys validation is to do standardised checks that every app would otherwise have to do itself. It's confusing to then test things that shouldn't be checked. Now that we know these aren't relevant to app validation, I agree with the decision to delete them